astro/src/content/json/applications/post-request.json

```diff
@@ -75,14 +76,18 @@
       "authorizedURLValidationPolicy": "ExactMatch",
       "clientAuthenticationPolicy": "Required",
       "clientSecret": "+fcXet9Iu2kQi61yWD9Tu4ReZ113P6yEAkr32v6WKOQ=",
+      "consentMode": "RememberDecision",
       "enabledGrants": [
         "authorization_code",
         "refresh_token"
       ],
       "generateRefreshTokens": true,
       "logoutBehavior": "AllApplications",
       "logoutURL": "http://www.example.com/logout",
-      "proofKeyForCodeExchangePolicy": "NotRequired"
+      "proofKeyForCodeExchangePolicy": "NotRequired",
+      "relationship": "ThirdParty",
```

Comment:
Would probably suggest this being `FirstParty` since that is the default. I guess that doesn't let you show the consent mode, then, though.